mobile_config/tests/integrations/migrations.rs

```diff
@@ -0,0 +1,67 @@
+use crate::common::{self, gateway_db::PreHistoricalGateway, partial_migrator::PartialMigrator};
+use chrono::{Duration, Utc};
+use helium_crypto::PublicKeyBinary;
+use mobile_config::gateway::db::{Gateway, GatewayType};
+use sqlx::PgPool;
+
+#[sqlx::test(migrations = false)]
+async fn gateways_historical(pool: PgPool) -> anyhow::Result<()> {
+    let partial_migrator = PartialMigrator::new(pool.clone(), vec![20251003000000], None).await?;
+
+    partial_migrator.run_partial().await?;
+
+    let address = pk_binary();
+    let now = Utc::now();
+    let one_min_ago = now - Duration::minutes(1);
+
+    let pre_gw = PreHistoricalGateway {
+        address: address.clone(),
+        gateway_type: GatewayType::WifiIndoor,
+        created_at: one_min_ago,
+        updated_at: now,
+        refreshed_at: now,
+        last_changed_at: now,
+        hash: "h0".to_string(),
+        antenna: Some(1),
+        elevation: Some(2),
+        azimuth: Some(3),
+        location: Some(123),
+        location_changed_at: Some(now),
+        location_asserts: Some(1),
+    };
+
+    pre_gw.insert(&pool).await?;
+
+    partial_migrator.run_skipped().await?;
+
+    let gw = Gateway::get_by_address(&pool, &address)
+        .await?
+        .expect("should find gateway");
+
+    println!("pre_gw: {:?}", pre_gw);
+    println!("gw: {:?}", gw);
+
+    assert!(pre_gw.address == gw.address);
```

Comment:
We should be using `assert_eq!` for all of these comparisons.

`assert!` gives us the code, but no hint as to what's wrong.
```rs
assert!(pre_gw.gateway_type == gw.gateway_type);
==> assertion failed: pre_gw.address == gw.address
```

`assert_eq!` gives us the failing values
```rs
assert_eq!(pre_gw.gateway_type, gw.gateway_type);
==> assertion `left == right` failed
==>   left: PublicKeyBinary { network: MainNet, type: Ed25519, address: "14QcXuwpGias5dxxxMzryBSPhA7qK8sbF2Se2wstsue82Ro6zCt" }
==>  right: PublicKeyBinary { network: MainNet, type: EccCompact, address: "112edB6q" }
```